user/common-tasks/backup-emergency-restore-v4.md

```diff
@@ -148,7 +148,7 @@ Emergency Recovery Instructions
  6. Verify the integrity of and decrypt the `private.img` file that houses your
     data.
 
-        [user@restore ~]$ for f_enc in vm1/private.img.???.enc; do \
+        [user@restore ~]$ find vm1 -name 'private.img.*.enc' | sort -V | while read f_enc; do \
```

Comment:
We need to qoute with double quotes `find vm1 -name "private.img.*[0-9].enc" | ...` and we should be consistent how substitute. In this case only numbers. With single quotes the find command does nothing.

Reply:
This shouldn't make any difference. This is a string passed to find who interpret it as a pattern. Not a pattern for shell.

Reply:
Yes, it intentionally avoids the shell's glob expansion.

I considered using `printf '%s\n' private.img.*[0-9].enc`, but if the rescue environment's shell isn't compiled with `printf` as a built-in, this would run into the command-line length limit.

Reply:
That works within the CLI (bash), but I script it to cater for private.img and root.img, hence I got a variable image_type, and therefore I had to replace the single quotes with double quotes. Otherwise the variable $image_type is hidden from the script and won't be resolved:

`(...)
echo ""
	echo "Encrypting the chunks."
	echo ""
	find $1 -name "$image_type.img.*[0-9].enc" | sort -V | while read f_enc
	do
                f_dec=${f_enc%.enc}
                echo "$backup_id!$f_dec!$backup_pass" | \
                        scrypt dec -P $f_enc $f_dec || break
	
		echo $f_enc
		echo $f_dec
        done
(...)
`
Sorry for the confusion. I should have said that I'm working on a script.